examples/todo/src/models/todo.model.ts

```diff
@@ -29,17 +29,6 @@ export class Todo extends Entity {
   })
   isComplete?: boolean;
 
-  @property({
-    type: 'string',
-  })
-  remindAtAddress?: string; // address,city,zipcode
-
-  // TODO(bajtos) Use LoopBack's GeoPoint type here
-  @property({
-    type: 'string',
-  })
-  remindAtGeo?: string; // latitude,longitude
-
```

Comment:
In order to implement geocoder integration while using `public-models/product.config.json`, we need to customize the controller method defined by `@loopback/rest-crud`. That's kind of against the spirit of this change, where we want to use the default REST API as provided by LB. At least for the initial release, I'd like us to recommend users to switch from `@loopback/rest-crud` to `lb4 repository` & `lb4 controller` if they need any tweaks in the built-in functionality.